src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/VacancyStatus.cshtml

```diff
@@ -1,24 +1,58 @@
-﻿@using SFA.DAS.EmployerAccounts.Web.Extensions
+﻿@using SFA.DAS.EAS.Portal.Client.Types;
 @model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
-
 @{
-    var vacancy = Model.AccountViewModel.SingleVacancy;
-}
+    Vacancy vacancy = null;
+
+    var actionLinkText = "";
+    int? numberOfApplications = null;
+    string status = "";
+
+    if (Model.HasSingleClosedVacancy)
```

Comment:
could the Row2COlumn2 action call a helper to create a "VacancyViewModel" which we could pass into this partial and therefore test this logic ?